backend/package.json

```diff
@@ -32,6 +32,33 @@
 				}
```

Comment:
It seems that command yeomanUI.toggleLog is not relevant.

Reply:
Done